src/atoms/toogle/styles.tsx

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import { rem } from 'polished';
+import { colors } from '../../ions/variables';
+
+const { light, primary, info } = colors;
+
+export const Switcher = styled.fieldset`
+  margin: 0;
+  border: none;
+  padding: 0;
+
+  > div {
+    label {
+      cursor: pointer;
+    }
+
+    .wrapper {
+      margin: 0 ${rem('10px')};
+      display: inline-block;
```

Comment:
```suggestion
    label {
      cursor: pointer;

      &:first-child {
        margin-right: ${rem('10px')};
      }

      &:last-child {
        margin-left: ${rem('10px')};
      }
    }

    .wrapper {
      display: inline-block;
```

This way the toggle doesn't have the margins even when it doesn't have the labels.